src/test/fixtures/fixtures_test.ts

```diff
@@ -118,14 +117,14 @@ suite('Fixtures', () => {
         continue;
       }
 
-      test(`packages/${fixtureBasename}`, async () => {
+      test(`packages/${fixtureBasename}`, async() => {
         const fixtureSourceDir = path.join(fixtureDir, 'source');
         const fixtureExpectedDir = path.join(fixtureDir, 'expected');
         const fixtureResultDir = path.join(fixtureDir, 'generated');
         const fixtureTestConfig = require(path.join(fixtureDir, 'test.js'));
         assert.isOk(fs.statSync(fixtureSourceDir).isDirectory());
         assert.isOk(fs.statSync(fixtureExpectedDir).isDirectory());
-        rimraf.sync(fixtureResultDir);
+        fs.copy(fixtureSourceDir, fixtureResultDir);
```

Comment:
isn't this async? Add `await`

Reply:
Done.

> Can you confirm that this runs in any reasonable amount of time? I believe the current tests diff all files across expected and generated, so I expect this adds a ton of work to compare the two untouched bower_components directories.

After the bower change, `npm run test:full` runs in 35 sec, exactly the same as running it on `master`.

> Regardless, add logic to ignore diffing bower_components & any other known-okay dirs

Done. We now don't copy any `bower_components` or `node_modules` files.